doc/technical.rst

```diff
@@ -209,6 +209,10 @@ workdir instead.
 |rarr| ``likelihood.``\ \*\ ``.ll``                 contains text of the last log likelihood value for an
                                                     instance. Segway uses this to decide when to
                                                     stop training
+``posterior``\ \*\ ``.bedGraph.gz``                 posterior probabilities for the given label in bedGraph 
+                                                    format
+``posterior``\ \*\ ``.track.gz``                    posterior probabilities for the given label in track
```

Comment:
"track format" probably should just be "bed file with trackline" instead effectively. It is more likely people will understand a BED file vs a .track file so describing it as is would probably be more helpful.